config/application.rb

```diff
@@ -38,6 +38,8 @@ class Application < Rails::Application
 
     config.elasticsearch = config_for(:elasticsearch)
 
+    config.ssl_options = { hsts: { expires: 1.week } }
```

Comment:
porting this over from `secure_headers` so that we keep only CSP-related logic in `secure_headers` and let Rails deal with what it can, as much as possible.